xep-0359.xml

```diff
@@ -122,6 +130,27 @@
 	<li>The value of the 'by' attribute MUST be the XMPP address of the entity assigning the unique and stable stanza ID. Note that XMPP addresses are normalized as defined in &rfc6122;</li>
   </ol>
 </section1>
+<section1 topic='Discovering Support' anchor='disco'>
+  <p>An entity that adds stanza IDs and follows the business rules, especially the rule on overriding the ID in elements where the by atttribute matches the entities own XMPP address, SHOULD announce the 'urn:xmpp:sid:0' namespace in its disco features allowing other entities to verify that those business rules are properly enforced.</p>
```

Comment:
I'd remove the "adds stanza IDs and" part. There could be entities which just enforce the rules but do not add stanza IDs.